yb-voyager/cmd/importData.go

```diff
@@ -828,15 +828,19 @@ func doOneImport(task *SplitFileImportTask, targetChan chan *tgtdb.Target) {
 
 			dbVersion := targetServer.DB().GetVersion()
 
-			for i, statement := range IMPORT_SESSION_SETTERS {
-				if checkSessionVariableSupported(i, dbVersion) {
-					_, err := conn.Exec(context.Background(), statement)
-					if err != nil {
-						utils.ErrExit("import file %q: run query %q on %q: %s", inProgressFilePath, statement, targetServer.Host, err)
+			sessionVarsPath := filepath.Join(exportDir, "metainfo", "ybSessionVars.sql")
+			if utils.FileOrFolderExists(sessionVarsPath) {
+				addCustomSessionVars(conn, sessionVarsPath)
+			} else {
+				for i, statement := range IMPORT_SESSION_SETTERS {
+					if checkSessionVariableSupported(i, dbVersion) {
+						_, err := conn.Exec(context.Background(), statement)
+						if err != nil {
+							utils.ErrExit("import file %q: run query %q on %q: %s", inProgressFilePath, statement, targetServer.Host, err)
+						}
```

Comment:
We should always run this loop first; and then execute the ybSessionVars.sql iff it exists.